server/src/presentation/socket/handlers/deleteRoom.ts

```diff
@@ -30,6 +35,31 @@ export function deleteRoomHandler({ io, socket, dataAccess }: HandlerConfig) {
                 );
             }
 
+            const avatarURLs = room.characters
+                .getCharacters()
+                .map(character => character.getProfileImageURL())
+                .filter(
+                    (url: string | undefined): url is string =>
+                        url !== undefined && url !== ''
+                );
+
+            for (const url of avatarURLs) {
+                await fileStorageProvider.deleteImageAtPublicURL(url);
+            }
+
+            const storyChunkImagesURLs = (
+                await room.story.fetchStory({ count: 9999 })
```

Comment:
Instead of magic number let's use at least some constant from common package, but event better solution would be implementing separate function for retrieving all image urls from story, should be an easy task and huge improvement